PyInstaller/hooks/hook-botocore.py

```diff
@@ -0,0 +1,3 @@
+from PyInstaller.utils.hooks import collect_data_files
+
+datas = collect_data_files('botocore')
```

Comment:
Please add a comment stating what botocore is, a link to it's homepage and which version you tested.